### PR TITLE
BigQuery - documentation edited (note ???)

### PR DESCRIPTION
**Pull request by alexander-levitski**

### CODE REVIEW

File: community/cloud-foundation/templates/bigquery/bigquery_dataset.py.schema [hunk 1]
```diff
@@ -144,18 +142,18 @@ outputs:
         description: The URI of the created resource.
     - etag:
         type: string
-        description: A has of the resource.
+        description: A has ??? hash? Or... ??? of the resource.
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/bigquery/bigquery_dataset.py.schema]: hash

File: community/cloud-foundation/templates/bigquery/examples/bigquery.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Example of the BigQuery Dataset and Table usage.
+# Example of the BigQuery (dataset and table) template usage.
```
Comment (sourced-vince) [inline community/cloud-foundation/templates/bigquery/examples/bigquery.yaml]: Any reaso why `(dataset and table)` is used here and not in the heading of the README? `This template creates a BigQuery dataset and table.`

Comment (alexander-levitski) [reply community/cloud-foundation/templates/bigquery/examples/bigquery.yaml]: Purely grammatical reason. An uninterrupted string of 5 words "BigQuery dataset and table template usage" is hard to parse for a human. A string of 3 words "BigQuery dataset and table" is not that hard to parse.
